src/Database/PG/Query/Class.hs

```diff
@@ -448,69 +451,105 @@ class ToPrepArgs a where
 instance ToPrepArgs () where
   toPrepArgs _ = []
 
+-- | Convert a value into a prepared query argument.
```

Comment:
This does what `toPrepVal` did, albeit with a slightly more helpful and consistent name?